server/services/gameCreate.js

```diff
@@ -141,6 +143,11 @@ module.exports = class GameCreateService {
             this.conversationService.createConversationAllPlayers(game);
         }
 
+        if (isCustomGalaxy) {
```

Comment:
It would be better to move this logic so that it is executed **before** the galaxy is generated, otherwise the user could add thousands of stars to their JSON and the server will generate the galaxy **and then** validate afterwards that the settings are correct.

Reply:
It completly ignores this setting and sets it to be correct, could just be put a limit to it, with Math.max or so

Reply:
The following bug mentioned above needs to be resolved:

1. On the create game screen.
2. Select "Stars Per Player" to be a value of 3.
3. Select "Custom" map type
4. Append JSON document.
5. Save -> **Validation error**.